gulpfile.js

```diff
@@ -27,6 +28,26 @@ if (/\.0$/.test(version)) {
     version = version.substring(0, version.length - 2);
 }
 
+var karmaConfigFile = path.join(__dirname, 'Specs/karma.conf.js');
+
+//Manually patch list of karma binary files until bug https://github.com/karma-runner/karma/issues/1070 is addressed
```

Comment:
I opened a pull request into karma to address the binary file issue: https://github.com/karma-runner/karma/pull/1816